crates/rapid-cli/src/commands/run.rs

```diff
@@ -175,7 +176,7 @@ fn handle_run_server(server_port: u16, app_type: String) {
 		s.succeed("Rapid build scripts installed!");
 	}
 
-	println!("{} Building rapid application...", rapid_logo());
+	info!("building rapid application...");
```

Comment:
Is there a reason for not using capitals? I understand that it makes things more minimal but it definitely feels a little odd :)

Reply:
That was mostly done just because NPM and Next does all lowercase as well, though I'm unsure if Vite or Remix does the same.

I'm not opposed to changing it back, it just looked out of place when it was the only uppercase letters in my output.

Reply:
I'm cool with lowercase!